classes/db/DbPDO.php

```diff
@@ -47,7 +47,7 @@ class DbPDOCore extends Db
      * @param int $timeout
      * @return PDO
      */
-    protected static function _getPDO($host, $user, $password, $dbname, $timeout = 5)
+    protected static function getPDO($host, $user, $password, $dbname, $timeout = 5)
```

Comment:
If you rename this function, you break backward compatibility. All public / protected functions can be overridden.

If you really want to keep this new function, please update the PR description to warn everybody about this.